app/views/service_calendars/master_calendar/pppv/template/_template_visit_input.html.haml

```diff
@@ -0,0 +1,23 @@
+-# Copyright © 2011 MUSC Foundation for Research Development
```

Comment:
From `_line_item_visit_input.html.haml`. Cleaned up and changed params as needed